calico/observability/view-flow-logs.mdx

```diff
@@ -3,4 +3,98 @@ description: View flow logs in the Calico Whisker web console.
+| `bytes_in`                        | long              | Number of incoming bytes since the last export.                                                                                                                                                                                                                                                                                                      |
+| `bytes_out`                       | long              | Number of outgoing bytes since the last export.                                                                                                                                                                                                                                                                                                      |
+| `dest_name`                       | keyword           | Contains one of the following values:<br />- Name of the destination pod.<br />- Name of the pod that was aggregated or the endpoint is not a pod. Check <code>dest_name_aggr</code> for more information, such as the name of the pod if it was aggregated.                                                                                         |
+| `dest_namespace`                  | keyword           | Namespace of the destination endpoint. A `-` means the endpoint is not namespaced.                                                                                                                                                                                                                                                                   |
+| `dest_port`                       | long              | Destination port. Not applicable for ICMP packets.                                                                                                                                                                                                                                                                                                   |
+| `dest_labels`                     | array of keywords | Labels applied to the destination pod. A hyphen indicates aggregation.                                                                                                                                                                                                                                                                               |
+| `reporter`                        | keyword           | - `src`: flow came from the pod that initiated the connection.<br />- `dst`: flow came from the pod that received the initial connection.                                                                                                                                                                                                            |
```

Comment:
```suggestion
| `reporter`                        | keyword           | - `src`: flow came from the source that initiated the connection.<br />- `dst`: flow came from the destination that received the initial connection.                                                                                                                                                                                                            |
```

Not all sources / destinations are pods. (e.g., host endpoints can appear in flow data)

Reply:
Changed.